src/bundle/Resources/public/js/alloyeditor/src/base/ez-embedimage.js

```diff
@@ -17,6 +17,7 @@ export default class EzEmbedImageButton extends EzEmbedDiscoverContentButton {
         const request = new Request(selection.item.ContentInfo.Content.ContentType._href, {
             method: 'GET',
             headers: {'Accept': 'application/vnd.ez.api.ContentType+json'},
```

Comment:
Should we also include `X-Siteaccess` and `X-CSRF-Token` headers?

Reply:
> X-CSRF-Token 

not needed for GET afaik

> X-Siteaccess

Yes, it is used in the ez-embed so should be used here to.


@sunpietro Should we abstract fetch a bit to avoid these kind of issues happens?

Reply:
Also, should we rather set `mode: 'same-origin'` instead of having to set `credentials: 'same-origin'` all over?

In which cases will we send REST request to another domain in these cases?